lib/headquarters/request.rb

```diff
@@ -8,8 +8,8 @@ class Request
     def self.perform_with_auth(http_method, path, params = {}, options = {})
       options_with_auth = options.merge(
         basic_auth: {
-          username: ENV['BASIC_AUTH_USER'],
-          password: ENV['BASIC_AUTH_PASS']
+          username: ENV['HQ_BASIC_AUTH_USER'] || ENV['BASIC_AUTH_USER'],
```

Comment:
because `BASIC_AUTH_USER` is too generic. you have to place this in your app's `.env`, where it is out of context

I kept the old version for retro compatibility. And since we're dropping this soon in favor of OAuth2, there's not much harm in that